### ISSUE TITLE
Translations missing + Admin does not work

### ISSUE THREAD

**Issue by JensDebergh** [SpinaCMS/Spina#805]
Hi!

I've been looking for a rails CMS and stumbled on SpinaCMS and gave it a go.

I'm running into multiple issues at the get go.

1) `en.spina.website.main` is missing a translation which already prevents me from going to the admin panel. It's an easy fix by just adding the missing translation since its just one.

Tip: Use `https://github.com/glebm/i18n-tasks` to make sure no translations are missing across multiple languages

2) Most of the actions in the admin panel, modals, navigation do not work and if they do work I'm getting turbowire results which aren't being processed correctly by the frontend.

I'm not sure if I'm doing anything wrong but I'm no newcomer to rails and I think there are some major blocking bugs in the latest version of Spina (I just decided to let you know, since maybe you weren't aware of the issue).

Video:

https://monosnap.com/file/5SjY1FQ9Q0xd68I1PhCBrb0kqUa4WC

**Comment by Bramjetten** [comment]
Thanks for noticing that missing translation, I've added it to the locales. I don't think Rails should raise an exception for missing translations, is that something you've changed?

**Comment by JensDebergh** [comment]
1) First I was using `3.0.2` but I downgraded to `2.7.1` to see if the issues still persist and they did.

2) My configuration is always set to raising exceptions when translations are missing. This way I'm never missing a translation since they entire rails app crashes.

3)
<img width="802" alt="Screenshot 2021-08-31 at 17 13 17" src="https://user-images.githubusercontent.com/6651506/131529807-a2d556de-d16f-4b5d-ac63-0951b5d84ec7.png">

- The rails console does not look like its doing anything.
- No requests are being sent. (websockets and xhr which may indicate the error happens even before)

**Comment by Bramjetten** [comment]
The error had to do with that first line in your js console. Did you try a different browser?

**Comment by getbizzey** [comment]
I haven't tried a different browser yet but shouldn't this just work on chrome?

**Comment by Bramjetten** [comment]
Yes, but there’s a couple of bugs with importmaps in Chrome. I’ll check it out later today and fix it!

**Comment by Bramjetten** [comment]
@JensDebergh Is it possible you're using an older version of Chrome? (older than v89?) Spina supports Chrome/Edge from v89+: https://caniuse.com/?search=importmap It uses an importmap shim for the others (Firefox/Safari)

**Comment by JensDebergh** [comment]
@Bramjetten Sorry for the delay. 

I'm on `Version 92.0.4515.159 (Official Build) (x86_64)` chrome. I've tried with firefox and I'm not experiencing the same issues like I have on chrome. 

Weird my chrome is bugged this way. Either way I can continue experimenting with SpinaCMS and see if its a good fit for my company.

Thanks for the help!

Groeten uit België ;)

**Comment by Bramjetten** [comment]
Awesome! Question: is the [live demo](https://spinacms-demo.herokuapp.com/admin/pages) broken for you in Chrome?

**Comment by JensDebergh** [comment]
Live demo works like a charm on chrome. I just noticed my local installation did not and that's why I opened the issue.
